lightningd/options.c

```diff
@@ -155,6 +155,19 @@ static char *opt_add_ipaddr(const char *arg, struct lightningd *ld)
 
 }
 
+static char *opt_add_localsocket(const char *arg, struct lightningd *ld)
```

Comment:
`listconfigs` is crashing in CI, probably due to not handling this case properly in `add_config` function. https://github.com/ElementsProject/lightning/blob/1a5514787037039477b27c7762461c5f2dde34a1/lightningd/options.c#L874-L877